examples/advanced/cugraph/rgcn.py

```diff
@@ -8,19 +8,20 @@
 code changes from the current `entity_sample.py` example.
```

Comment:
Did you see similar performance numbers after running this script?

Reply:
Yes, performance is the same as before.